webApp.js

```diff
@@ -1,18 +1,16 @@
 import express from 'express';
-import {AsyncLocalStorage, executionAsyncId} from 'async_hooks';
+import {storage} from './storage.js'
 import * as uuid from 'uuid';
 import router from './router.js'
 
 let app = express();
-let storage = new AsyncLocalStorage();
 
 global.kResourceStore = storage.kResourceStore;
 
-app.use(async (req, res, next) => {
-    await storage.run(new Map(), () => {
+app.use((req, res, next) => {
```

Comment:
never write an `async` express middleware. There is nothing to handle promise rejection at that level

Reply:
async here, it's experiment for make it work

Reply:
Well, the `await` call later is not needed